src/CalcManager/CalculatorManager.cpp

```diff
@@ -613,9 +613,9 @@ namespace CalculationManager
         }
     }
 
-    wstring CalculatorManager::GetResultForRadix(uint32_t radix, int32_t precision)
+    wstring CalculatorManager::GetResultForRadix(uint32_t radix, int32_t precision, bool groupDigitsPerRadix)
```

Comment:
>wstring CalculatorManager::GetResultForRadix(uint32_t radix, int32_t precision, bool groupDigitsPerRadix) [](start = 4, length = 105)

It looks like we don't have tests that covered this before...  Can you add a test or two to sanity check this method(as well as your addition to it)?